examples/sample-pub/service/sample-pub-oauth-app.js

```diff
@@ -239,18 +239,20 @@ app.post('/token', (req, res) => {
 /**
  * Authorization sync endpoint.
  * Query parameters:
- * - publisher_id={publisher_id}
+ * - publication_id={publication_id}
  * - access_token={access_token}
  */
 app.all('/entitlements', (req, res) => {
-  const publisherId = getParam(req, 'publisher_id') ||
-      getParam(req, 'publication_id') || 'none';  // MIGRATE
+  const publicationId = getParam(req, 'publication_id') ||
+      getParam(req, 'publisher_id') ||  // MIGRATE
```

Comment:
Do we still need `getParam(req, 'publisher_id')`?

Reply:
Until we migrate completely.